src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/EventOutcome.java

```diff
@@ -4,10 +4,11 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
-@Data
+@Data()
```

Comment:
Are the parentheses required?

Reply:
No, part of me debugging. The Attendance model has them so I put them in just in case.